src/backup/BackupComplete.tsx

```diff
@@ -29,7 +31,12 @@ function BackupComplete({ route }: Props) {
   useEffect(() => {
     const timer = setTimeout(() => {
       if (navigatedFromSettings) {
-        navigate(Screens.Settings, { promptConfirmRemovalModal: true })
+        navigate(
+          getFeatureGate(StatsigFeatureGates.USE_TAB_NAVIGATOR)
+            ? Screens.Settings
+            : Screens.SettingsDrawer,
```

Comment:
I believe this will lead to a crash if a feature gate changes when the app is backgrounded. Can you test the follow scenario?
1. Have the feature on for your wallet, navigate to this screen (or any of the below screens) 
2. Background the app
3. Change your wallet to be on the false variant and give a few mins for things to be updated
4. Foreground the app and invoke the action to go to settings

Reply:
![Simulator Screenshot - iPhone 14 Pro - 2024-03-19 at 10 31 36](https://github.com/valora-inc/wallet/assets/140328381/07535cbc-2e8b-4e84-bc43-0a0cff09561f)
@satish-ravi you're right, I get this error when following this procedure to go back to settings. Is there a way to do navigate to make sure the screens are available, or should it just navigate home?

Reply:
you can navigate to Screens.DrawerNavigator with initial screen name as Screens.SettingsDrawer, there should be examples of others.